webui/react/src/components/DeterminedAuth.tsx

```diff
@@ -37,7 +35,11 @@ const DeterminedAuth: React.FC<Props> = ({ canceler, source }: Props) => {
     setCanSubmit(false);
     try {
       const options = { signal: canceler.signal };
-      const { token } = await login({ ...creds as Credentials, cancelToken: source.token });
+      const loginRequest: V1LoginRequest = {
+        password: creds.password || '',
```

Comment:
I thought maybe this might be better be moved under the `services/apiConfig` but then we'd need to define a new type other that v1LoginRequest

Reply:
Maybe I'm wrong but I think passing the `signal` as a `login` prop doesn't work?

Probably should be fixed like this:
<img width="755" alt="Schermata 2021-04-21 alle 16 07 01" src="https://user-images.githubusercontent.com/5413702/115567544-a70a8280-a2bb-11eb-98d4-0a06cd3646b3.png">